FYI: fix PR 11098
http://sourceware.org/ml/gdb-patches/2010-03/msg00117.html
http://sourceware.org/ml/gdb-cvs/2010-03/msg00020.html

### src/gdb/ChangeLog	2010/03/03 18:05:01	1.11418
### src/gdb/ChangeLog	2010/03/03 18:32:43	1.11419
## -1,3 +1,9 @@
+2010-03-03  Tom Tromey  <tromey@redhat.com>
+
+	PR mi/11098:
+	* varobj.c (install_new_value): Handle case where new print_value
+	is NULL.
+
 2010-03-03  Dainis Jonitis  <jonitis@gmail.com>
 
 	PR gdb/11345:
--- src/gdb/varobj.c	2010/02/23 17:01:54	1.155
+++ src/gdb/varobj.c	2010/03/03 18:32:44	1.156
@@ -1593,7 +1593,10 @@
     {
       xfree (print_value);
       print_value = value_get_print_value (var->value, var->format, var);
-      if (!var->print_value || strcmp (var->print_value, print_value) != 0)
+      if ((var->print_value == NULL && print_value != NULL)
+	  || (var->print_value != NULL && print_value == NULL)
+	  || (var->print_value != NULL && print_value != NULL
+	      && strcmp (var->print_value, print_value) != 0))
 	changed = 1;
     }
   if (var->print_value)
